kmp-library/build.gradle

```diff
@@ -50,8 +50,28 @@ kotlin {
             }
         }
     }
+    task packForXCode(type: Sync) {
```

Comment:
We need this task to generate an xcode framework we can import into the sample ios project.